### PR TITLE
Library dropdown fix

### PR DESCRIPTION
**Pull request by scovitin-vsevolod**
update db require

### CODE REVIEW

File: docroot/sites/all/modules/chm/ptk_base/forms/base.inc [hunk 1]
```diff
@@ -131,6 +131,25 @@ function ptk_base_form_alter(&$form, &$form_state, $form_id) {
   if (strpos($form_id, '_node_form')) {
     ptk_base_add_domain_menu($form, $form_state);
   }
+
+  if ($form_id == 'views_exposed_form') { // todo add domain check
+    $query = db_select('field_data_field_document_type', 'dt')
```
Comment (cristiroma) [inline docroot/sites/all/modules/chm/ptk_base/forms/base.inc]: @scovitin-vsevolod here is my feedback

* I don't like this code written inline - **not testable** . Please use the PTK class to create an API function to retrieve the terms is use and add the call here. `getDocumentsInUse` or something
* solve `todo`

File: docroot/sites/all/modules/chm/ptk_base/ptk_base.install [hunk 2]
```diff
@@ -33,4 +33,24 @@ function ptk_base_update_7002() {
...
+    ->entityCondition('bundle', array('document_type'));
+
+  $result = $query->execute();
+  $terms = array_keys($result['taxonomy_term']);
+  foreach($terms as $tid) {
+    $term = taxonomy_term_load($tid);
+    if ($term->name_field) {
+      continue;
+    }
+    $term->name_field[$term->language][0]['value'] = $term->name;
```
Comment (cristiroma) [inline docroot/sites/all/modules/chm/ptk_base/ptk_base.install]: How did the `name_field` end-up empty? Here's is handled the effect, I wonder which is the cause? Maybe those terms were created during a migration with a bug in it? Can you double-check please? Give some examples.

Comment (scovitin-vsevolod) [inline docroot/sites/all/modules/chm/ptk_base/ptk_base.install]: I haven't found scripts that created 'document_type' term. migration script only gets items

Comment (cristiroma) [inline docroot/sites/all/modules/chm/ptk_base/ptk_base.install]: The terms created by 'migrate' module are not multilingual :( We need to create the term manually and add the 'tid' as reference.

**Pull request merged**